scripts/migrate/cli.ts

```
@@ -0,0 +1,181 @@
+
+  return {
+    exitCode: report.blockers.length > 0 ? 1 : 0,
+    filesScanned: files.length,
+    findings,
+    blockers: report.blockers,
+    applied
+  };
+}
+
+const invokedDirectly =
+  process.argv[1] !== undefined && import.meta.url.endsWith(process.argv[1].split('/').pop() ?? '');
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)